test.py

```diff
@@ -0,0 +1,48 @@
+import pygame
```

Comment:
Is this test file intentionally added here?

Reply:
lmao